desdeo/mcdm/pareto_navigator.py

```diff
@@ -0,0 +1,271 @@
+"""Functions related to the Pareto Navigator method are defined here.
+
+Reference of the method:
+
+Eskelinen, Petri, et al. "Pareto navigator for interactive nonlinear
+multiobjective optimization." OR spectrum 32 (2010): 211-227.
+"""
+
+import numpy as np
+from scipy.optimize import linprog
+from scipy.spatial import ConvexHull
+
+from desdeo.problem import (
+    Problem,
+    get_ideal_dict,
+    get_nadir_dict,
+    numpy_array_to_objective_dict,
+    objective_dict_to_numpy_array
+)
+
+
+def classification_to_reference_point(
+    problem: Problem,
+    pref_info: dict[str, list[str]],
+    current_solution: dict[str, float]
+) -> dict[str, float]:
+    """Convert preference information given as classification into a reference point.
+
+    Args:
+        problem (Problem): The problem being solved.
+        pref_info (dict): The preference information given as classification.
+        current_solution (dict[str, float]): The current solution.
+
+    Returns:
+        dict[str, float]: A reference point converted from classification.
+    """
+    ref = []
+    ideal = objective_dict_to_numpy_array(problem, get_ideal_dict(problem))
+    nadir = objective_dict_to_numpy_array(problem, get_nadir_dict(problem))
+    sol = objective_dict_to_numpy_array(problem, current_solution)
+
+    for i in range(len(pref_info["classification"])):
```

Comment:
We should avoid referring to the objectives with an index. We can consider that `pref_info` is in the following format:

```python
pref_info = {"f_1": "<", "f_2": ">", "f_3": "="}
```

This way it is always explicitly clear to which objective function each classification belongs to. This allows to easily refer to the correct ideal value as well. E.g., by using the methods [`Problem.get_ideal_point()`](https://desdeo.readthedocs.io/en/desdeo2/api/desdeo_problem/#desdeo.problem.schema.Problem.get_ideal_point) and [`Problem.get_nadir_point()`](https://desdeo.readthedocs.io/en/desdeo2/api/desdeo_problem/#desdeo.problem.schema.Problem.get_nadir_point).

Can you adjust this function to be compliant with the above suggestion?

Reply:
Muutin tämän nyt niin, että tuossa pref_info argumentissa tulee dictinä nuo luokittelut.